portfolio/src/main/webapp/index.html

```diff
@@ -2,12 +2,12 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio.</title>
+    <title>Cindy Hao</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Should we load jquery before script.js? It seems like script.js depends on jquery.

You might find this interesting to read:
https://www.html5rocks.com/en/tutorials/speed/script-loading/

The concept of script loading and execution order is critical for frontend performance (and our team does a lot of work in this area :))

Reply:
OH wow ok I didn't know that! thanks for letting me know!